websites/K/Kick/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"name": "IPEXA",
+		"id": "642807365695176724"
+	},
+	"service": "Kick",
+	"description": {"en": "Kick: The new Streaming service, that offers you many different Streaming Content"},
+	"url": [
+		"kick.com",
+		"help.kick.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/kVabJjN.png",
+	"thumbnail": "https://i.imgur.com/HLui7QQ.png",
```

Comment:
Please upload a new screenshot for the thumbnail where the website is in English.

Or use: (Which IMO is a lot better and an actual thumbnail)
![image](https://github.com/PreMiD/Presences/assets/34792838/8b89dca4-ee89-4e08-abcd-937252ff2999)